### PR TITLE
add home page features, and several other changes

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add home page "features" with image and text
- rewrite some of the summary text on the research, resources, and team pages
- add more tools to resource page
- rework some includes
- refactor and rework some css
- rearrange contact page
- add funding images
- add upenn logo
- add resource thumbnail placeholder/fallback
- add summary text to resources page, because all of the other pages have one, and there are some things to say there about our approach to developing tools
- add header icons to join and collaborators pages

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,4 +6,46 @@ home: true
...
+We use data science to answer important questions in biology and medicine.
+Our approach to research prioritizes transparency, rigor, and reproducibility.
+
+[See all of our research &nbsp;→](research)
+{% endcapture %}
+{%
+  include feature.html
+  image="background-laptop-paper.jpg"
+  url="research"
+  heading="Data + biology = insight"
```
Comment (cgreene) [inline index.md:21]:
```suggestion
  heading="biological data + machine learning → insight"
```

File: index.md [hunk 2]
```diff
@@ -6,4 +6,46 @@ home: true
 The Greene Lab is a team of researchers at the [Perelman School of Medicine](https://www.med.upenn.edu/) at the [University of Pennsylvania](https://www.upenn.edu/) dedicated to answering important questions in biology and medicine with computation.
 We focus on bringing together publicly available [big data](https://en.wikipedia.org/wiki/Big_data), developing new computational methods to analyze that data, and creating tools to put those resources into the hands of every biologist.
 
-{% include figure.html url="team/group-photo-1.jpg" description="The Greene Lab team" full="true" %}
\ No newline at end of file
+---
+
+{% capture text %}
+We use data science to answer important questions in biology and medicine.
+Our approach to research prioritizes transparency, rigor, and reproducibility.
```
Comment (cgreene) [inline index.md:13]:
```suggestion
We prioritize transparency, rigor, and reproducibility in all of our research.
```

File: index.md [hunk 3]
```diff
@@ -6,4 +6,46 @@ home: true
...
+
+[See all of our research &nbsp;→](research)
```
Comment (cgreene) [inline index.md:15]:
```suggestion
[See what we've published &nbsp;→](research)
```

File: index.md [hunk 4]
```diff
@@ -6,4 +6,46 @@ home: true
...
+{%
+  include feature.html
+  image="background-laptop-paper.jpg"
+  url="research"
+  heading="Data + biology = insight"
+  text=text
+%}
+
+{% capture text %}
+All of our software, tools, datasets, etc. are 100% open-source.
```
Comment (cgreene) [inline index.md:26]:
```suggestion
We aim to make all of the software, tools, datasets, and other resources that we generate freely available and openly licensed.
```

File: index.md [hunk 5]
```diff
@@ -6,4 +6,46 @@ home: true
...
+We believe in creating free and transparent resources that anyone around the world can use.
```
Comment (cgreene) [inline index.md:27]:
```suggestion
We believe that anyone around the world should be able to replicate and extend our work.
```

File: index.md [hunk 6]
```diff
@@ -6,4 +6,46 @@ home: true
...
+
+[Check out our useful resources &nbsp;→](resources)
```
Comment (cgreene) [inline index.md:29]:
```suggestion
[See our highlighted resources &nbsp;→](resources)
```

File: index.md [hunk 7]
```diff
@@ -6,4 +6,46 @@ home: true
...
+  url="resources"
+  heading="Open source, all the way"
+  text=text
+%}
+
+{% capture text %}
+Our lab is a team of a highly engaged and collaborative team of researchers that want to push the frontier of data science.
+We recognize that diversity of experience and perspective is key in the process of discovery.
+
+[Learn more about our team &nbsp;→](team)
```
Comment (cgreene) [inline index.md:43]:
```suggestion
[Meet our team &nbsp;→](team)
```

File: index.md [hunk 8]
```diff
@@ -6,4 +6,46 @@ home: true
...
+{%
+  include feature.html
+  image="code-photo.jpg"
+  url="resources"
+  heading="Open source, all the way"
+  text=text
+%}
+
+{% capture text %}
+Our lab is a team of a highly engaged and collaborative team of researchers that want to push the frontier of data science.
```
Comment (cgreene) [inline index.md:40]:
```suggestion
We are a team of enthusiastic researchers who want to push the frontier of data science.
```

File: index.md [hunk 9]
```diff
@@ -6,4 +6,46 @@ home: true
...
+We recognize that diversity of experience and perspective is key in the process of discovery.
```
Comment (cgreene) [inline index.md:41]:
```suggestion
We strive to build an inclusive environment for research and recognize that using artificial intelligence techniques to improve equity will require diverse teams.
```

File: research/index.md [hunk 10]
```diff
@@ -4,14 +4,13 @@ icon: fas fa-microscope
 redirect_from: /publications
 ---
 
-There are millions of samples that have been subjected to genome-wide profiling and these data are freely downloadable.
-Our lab develops algorithms that integrate these data to help us understand how biological systems work.
-The breadth of these data help us probe many different biological conditions, including those with limited data, such as rare diseases.
-We recognize that our lab won't have all the answers, or even all of the questions, so we aim to develop systems that any biologist can use to interpret these data.
-We also strive to make our workflows robust and reusable for other computational scientists.
-Our open, rigorous and transparent approach to research facilitates such reuse.
+There is an abundance of publicly available data about various biological systems, but the data are often hard to make useful conclusions out of.
```
Comment (cgreene) [inline research/index.md:7]:
```suggestion
There is an abundance of publicly available data measuring various biological systems, but often it remains difficult to draw insight from individual experiments or datasets.
```

File: team/index.md [hunk 11]
```diff
@@ -10,7 +10,8 @@ redirect_from:
 ---
 
 Our lab is made up of a highly engaged and collaborative team of researchers.
-We recognize the value of diversity in the process of discovery, and we foster an environment where team members are treated equally and we respect and admire our differences.
+We recognize that diversity of experience and perspective is key in the process of discovery.
```
Comment (cgreene) [inline team/index.md:13]:
```suggestion
We recognize that diverse teams do better research.
```

File: team/index.md [hunk 12]
```diff
@@ -47,6 +48,21 @@ They have all made lasting contributions to science and to our hearts. ❤️
...
+{%
+  include multi-figure.html
+  image1="team/alfred-p-sloan-foundation.png"
+  description1="Alfred P Sloan Foundation"
+  url1="https://www.sloan.org/"
+  image2="team/gordon-betty-moore-foundation.png"
+  description2="Gordon and Betty Moore Foundation"
+  url2="https://www.moore.org/"
+%}
+
```
Comment (cgreene) [inline team/index.md:65]: How many can this fit? We also should add CZI, CF Foundation, ALSF, NHGRI, NCI, NINDS, NHLBI

**Review by cgreene — APPROVED** [review]
A few thoughts! Much excitement on seeing this launch :)

File: team/index.md (hunk 12 above)
Comment (vincerubinetti) [reply team/index.md:65]: I'd recommend only 4 per row. Are these in order of importance?

Comment (cgreene) [reply team/index.md:65]: Row 1:
* G&B Moore
* NCI
* NHGRI
* ALSF

Row 2:
* CZI
* CF Foundation
* NHLBI
* NINDS

Comment (cgreene) [reply team/index.md:65]: Sloan is a row 2 or row 3 :) we could move any of row 2 to row 3

Comment (vincerubinetti) [reply team/index.md:65]: No Sloan?

**Pull request merged**